mozilla_bitbar_devicepool/configuration.py

```diff
@@ -81,6 +99,7 @@ def configure(bitbar_configpath, filespath=None, update_bitbar=False):
 
     with open(bitbar_configpath) as bitbar_configfile:
         CONFIG = yaml.load(bitbar_configfile.read(), Loader=yaml.SafeLoader)
+    ensure_filenames_are_unique(CONFIG)
```

Comment:
What will happen here if the ConfigurationFileDuplicateFilenamesException or ConfigurationFileException are raised?

Reply:
The exception should be fatal and the program will exit (we haven't started threads yet at this point). 

Modified to handle the exceptions more gracefully.